libs/application/templates/health-insurance/src/lib/HealthInsuranceTemplate.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  ApplicationTemplate,
+  ApplicationTypes,
+  ApplicationContext,
+  ApplicationRole,
+  ApplicationStateSchema,
+} from '@island.is/application/core'
+import * as z from 'zod'
+
+type Events =
+  | { type: 'APPROVE' }
+  | { type: 'REJECT' }
+  | { type: 'SUBMIT' }
+  | { type: 'ABORT' }
+
+const HealthInsuranceSchema = z.object({
+  personalDataText: z.string(),
+  occupationText: z.string(),
+  infoInput: z.string(),
+  summaryInput: z.string(),
+})
+
+const HealthInsuranceTemplate: ApplicationTemplate<
+  ApplicationContext,
+  ApplicationStateSchema<Events>,
+  Events
+> = {
+  type: ApplicationTypes.EXAMPLE,
```

Comment:
here you have to use the `ApplicationTypes.HEALTH_INSURANCE`

Reply:
Great you spotted it, had missed this. Updated it 👍